class/defaults.yml

```diff
@@ -3,6 +3,27 @@ parameters:
     =_metadata: {}
     namespace: appuio-cloud
 
+    images:
+      agent:
+        registry: ghcr.io
+        repository: appuio/appuio-cloud-agent
+        tag: v0.1.2
+
+    agent:
+      replicas: 3
+      resource-ratio:
```

Comment:
Do those keys need to be kebab-cased? We already have a mix of every casing imaginable 🙈

Reply:
Fair point :sweat_smile: